extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -51,6 +63,21 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
    */
   constructor(element) {
     super(element, InteractiveType.SLIDER, [0, 1]);
+    this.sliderValue = null;
+    this.inputSlider = null;
```

Comment:
We'll need comments for this, you can check with `amp pr-check --nobuild` that it will probably give you an error saying all private variables need comments. Also, private variables (and functions) end with `_`, so you'd call them `this.bubbleEl_` (I think `bubbleEl_` and `inputEl_` are better names since the slider is the component itself, and `El` means it's an element)

The comments for each will probably be sth like:

```js
/* @private {?Element} bubble containing the current selection of the user while dragging */
this.bubbleEl_ = null;
```

you can check on the other files to see how the private variables are defined in the constructor for more reference